### PR TITLE
Feat/home page

### PR DESCRIPTION
**Pull request by tonymike99**
Hi team,
I've added the home page for the quiz application.

https://deploy-preview-1--quizothrill.netlify.app/

[Check only these files](https://github.com/tonymike99/quizothrill/pull/1/files?file-filters%5B%5D=.css&file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&show-viewed-files=true)

### CODE REVIEW

File: src/assets/css/styles-common.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+.color-white {
+  color: var(--light-color);
+}
+
+.card-category {
+  border: 5px solid var(--light-color);
+}
+
+.position-modified {
+  left: 47% !important;
```
Comment (amartya-rs) [inline src/assets/css/styles-common.css:109]: suggestion : we should avoid using important

File: src/components/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+function Card({ category }) {
```
Comment (amartya-rs) [inline src/components/Card.jsx:1]: suggestion: use ES6 arrow functions

**Review by amartya-rs — APPROVED** [review]
Good work Tony.
The code is well commented. And the slide show on the homepage looks awesome.

**Review by Shubhamkr2610 — APPROVED** [review]
Apprecition : good way to give message for css for every particular  items. It will help to understanding the code and css UI

File: src/components/index.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export { Slideshow } from "./Slideshow";
+export { Header } from "./Header/Header";
+export { Footer } from "./Footer/Footer";
+export { Card } from "./Card";
```
Comment (Shubhamkr2610) [inline src/components/index.js:4]: Appreciation: good way of exporting files .

**Review by Shubhamkr2610 — APPROVED** [review]
just made the correction that other folks suggested , then you are good to go

File: src/assets/css/styles-common.css [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+.styled-link,
+.styled-link-2 {
+  color: inherit;
+}
+
+.styled-link:hover,
+.styled-link-2 > i:hover {
+  background-color: #36096d;
+  background-image: linear-gradient(315deg, #36096d 0%, #37d5d6 80%);
```
Comment (varun13169) [inline src/assets/css/styles-common.css:68]: Can the colors be parameterized?

File: src/components/Header/Header.css [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+.header-item {
+  margin: 1rem;
+}
+
+.list-navbar {
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-items: center;
+  gap: 4rem !important;
```
Comment (varun13169) [inline src/components/Header/Header.css:36]: Do not use Important

File: src/components/Header/Header.css [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+.badge-outside-top-right {
+  top: -25% !important;
```
Comment (varun13169) [inline src/components/Header/Header.css:40]: Do not use Important

File: src/components/Header/Header.css [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+  right: -15% !important;
```
Comment (varun13169) [inline src/components/Header/Header.css:41]: Do not use Important

**Pull request merged**